backend/session 4/app.py

```diff
@@ -0,0 +1,207 @@
+from flask import Flask, request, session, jsonify
+from models import db, User, Task, Note ,Abroad_blogs
+from flask_cors import CORS
+
+app = Flask(__name__)
+
+CORS(app)
+
+app.config["SQLALCHEMY_DATABASE_URI"] = "sqlite:///databaeworks.db"
+db.init_app(app)
+
+with app.app_context():
+    db.create_all()
+
+CURRENT_USER = "" 
+
+
+@app.route("/signup", methods=["POST"])
+def registration():
+    json = request.get_json()
+    user = User(
+        email=json["email"],
+        password=json["password"],
+        username=json["username"],
+        age=json["age"],
+        school=json["school"],
+        gender=json["gender"],
+    )
+
+    user_db_email = User.query.filter_by(email=user.email).first()
+    user_db_email_username = User.query.filter_by(username=user.username).first()
+    if len(user.password) < 6:
+        return "Password is short"
+    if not (user_db_email is None):
+        return "Email already saved"
```

Comment:
You need to return a json body for frontend, Also specify the status code returned, because otherwise it's considered 200,
Here: `409 (Conflict) is returned, the email address already exists.`
To return with a code, basically try: return JSONBODY, 409 (for example) 

Checkout [Status Codes](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status)